sszgen/main.go

```diff
@@ -349,7 +357,7 @@ func (e *env) hashSource() (string, error) {
 	return hex.EncodeToString(hash[:]), nil
 }
 
-func (e *env) print(first bool, order []string, experimental bool) (string, bool, error) {
+func (e *env) print(order []string, experimental bool) (string, bool, error) {
```

Comment:
I removed this boolean from the function signature because the value was unused by the function body.